radio/src/translations/cz.h

```diff
@@ -743,6 +743,7 @@
 #define TR_DELETE_THEME                "Smazat motiv?"
 #define TR_SAVE_THEME                  "Uložit motiv?"
 #define TR_EDIT_COLOR                  "Edit Color"
```

Comment:
Upravit barvu

Reply:
This is different to the translation you posted for #3278  (Změnit barvu)

Reply:
Both are right... Czech language has many synonyms :) 

Please leave "Upravit barvu"